package/MDAnalysis/coordinates/base.py

```diff
@@ -1545,6 +1578,61 @@ def has_valid_coordinates(self, criteria, x):
         x = np.ravel(x)
         return np.all(criteria["min"] < x) and np.all(x <= criteria["max"])
 
+class SingleFrameWriterBase(WriterBase):
+    """Base class for single frame writers.
+
+    See :ref:`Trajectory API` definition in for the required attributes and
+    methods.
+
+    .. versionadded:: 3.0.0
+       Separate single frame writer as new base class.
+    """
+    def __init__(self,
+                 filename,
+                 n_atoms,
+                 append=False):
+        """Set up single frame writer.
+
+        Parameters
+        ----------
+        filename : str
+            filename of trajectory file
+        n_atoms : int
+            number of atoms in trajectory
+        append: bool (optional)
+            If ``True``, append to an existing file. If ``False``, overwrite
+            the file. Default is ``False``.
```

Comment:
Document that this option only exists for API compatibility and will raise an issue